client/trino-jdbc/src/main/java/io/trino/jdbc/TrinoConnection.java

```diff
@@ -24,6 +24,8 @@
 import io.trino.client.ServerInfo;
 import io.trino.client.StatementClient;
 
+import javax.annotation.Nullable;
```

Comment:
in cmt msg

> When Statement or Connection is abandoned there is no point
to continue query execution.

abandoned -> closed

to me, "abandoned" means more like "forgotten", unreferenced, garbage collected